app/views/shared/_navbar.html.erb

```diff
@@ -15,16 +15,15 @@
           <%= link_to "All Listings", listings_path, class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
-          <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          <%= image_tag "sakura-photo.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-            <%= link_to "Action", "#", class: "dropdown-item" %>
-            <%= link_to "Another action", "#", class: "dropdown-item" %>
+            <%= link_to profile_path(current_user), class: "dropdown-item" do %>
+              👋 Welcome <%= current_user.first_name %>
+            <% end %>
+            <%= link_to "Bookings", "#", class: "dropdown-item" %>
```

Comment:
Should this be: 
`<%= link_to "Bookings", bookings_path, class: "dropdown-item" %>` ?

Reply:
@worksofgenius yes but same as above, I was thinking whether to let the user check all their bookings on their profile page or go to a separate booking details page, so I temporarily put it as `#` 😅